projects/components/src/filter-tabs/filter-tabs.component.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  QueryList,
+  ViewChildren
+} from '@angular/core';
+import { TypedSimpleChanges } from '@hypertrace/common';
+import { FilterTab } from './filter-tab';
+import { FilterTabComponent } from './filter-tab.component';
+
+@Component({
+  selector: 'htc-filter-tabs',
+  styleUrls: ['./filter-tabs.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="filter-tabs" *ngIf="this.activeTab">
+      <div
+        class="active"
+        *ngIf="this.activeElementPosition"
+        [style.left.px]="this.activeElementPosition.left"
+        [style.width.px]="this.activeElementPosition.width"
+      ></div>
+      <div class="container" *ngFor="let tab of this.tabs; let index = index">
+        <div class="divider" *ngIf="index !== 0" [class.adjacent-active]="this.isAdjacentActiveTab(index)"></div>
+        <htc-filter-tab class="tab" [label]="tab.label" (click)="this.setActiveTab(tab)"></htc-filter-tab>
+      </div>
+    </div>
+  `
+})
+export class FilterTabsComponent implements AfterViewInit, OnChanges {
```

Comment:
Is this going to replace toggle filter?

Reply:
Toggle Button? No, this component is specifically for toggle filters. The toggle buttons will still exist but won't be used for filters anymore.